### DISCUSSION TITLE
report following restart

### DISCUSSION THREAD

**Discussion by osramini** [osmandapp/OsmAnd#18234] [Q&A]
hello everyone, I just posted here to get some information regarding the reports that the Osmand application generates following a crash at startup.
when I start Osmand+, it sometimes happens that the application stops and asks me to send a report, I tried to open this report with a notepad, but I do not understand anything that is in this file, I would like to understand why Osmand stops so often and how to solve this problem,
thank you for your information.

**Comment by DmitryAlexei** [comment]
@osramini Please update OsmAnd to the latest version from the Play market or try beta or nightly build (https://osmand.net/docs/versions/nightly_versions) and see if your issue persists.
If you still encounter this error, you can add crash log here and someone from our community will take a look.

**Reply by osramini** [reply]
Hello everyone,
thank you DMITRYALEXEI for this quick response,
As for updates, I monitor them very regularly on my S23ultra, I am currently on 4.5.10,
my last crashes that I had were at startup, I cut the network connections so that nothing disrupts the phone, it starts up fine and I get this error message "oops Osmand crashed", I leave it though the application start, I also have these crashes along the way, if I am in vehicle mode in a car and I select a track it saves and when I have the crash it still stays recording, I just have to start Osmand again,
I also noticed that if I am in "pedestrian" mode and I drive by car, the routes are in a straight line and cut "through the houses", in the car I have to select the "car" mode,
and when I want to turn off the screen to save battery I also have these crashes,
a little desperate, I "finally" managed to master this application thinking I could replace my Montana 700, but these constant crashes are frustrating.
I'm going to try it again and the next time it crashes, I'll post the report.

**Reply by osramini** [reply]
I have other apps on my S23Ultra, unfortunately I don't have a phone without apps to try...

**Reply by osramini** [reply]
I did several tests again today, it was not very good, several crashes, the start was good, as I had to drive, I put my "car" profile to avoid having the tracks that cut above houses, as described previously,
I had the application close, I stop to restart the application, I drive for a while and it starts again, the application closes completely, but once restarted, the recording of the trace is always recorded, each time I have a crash report, which I recorded, I attach a copy, I edited it with a notepad, I have a few lines where I think I can read that I I have memory errors, but why? memory problems with an S23 Ultra and 12 gigs of memory?
too bad, I had managed to master this application well which had put me off a little at the start as it was so complex, and here are these additional problems, not to mention that my "other" concern concerns the screen, not that of the S23 Ultra which is splendid and gives me a very good rendering of the Osmand+ cartography, but this screen which in direct sunlight is totally illegible, next to my Montana 700 it's the opposite, in direct sunlight is very readable, plus there has sun and the more it is readable, less beautiful when the light is lower,
I tried to add brightness as the S23U allows, but to no avail, when a beautiful sun shines, the Montana 700 is the most readable....
thank you for those who can already inform me about this problem of Osmand which closes by

**Comment by osramini** [comment]
[exception.txt](https://github.com/osmandapp/OsmAnd/files/12742216/exception.txt)
